app/models/service_template_ansible_playbook.rb

```diff
@@ -104,7 +104,7 @@ def self.build_parameter_list(name, description, info)
       :ask_limit_on_launch      => true,
       :ask_inventory_on_launch  => true,
       :ask_credential_on_launch => true
-    }
+    }.merge(info.slice(:become_enabled))
```

Comment:
This way we can add more options easily, for example `info.slice(:become_enabled, :allow_simultaneous)`